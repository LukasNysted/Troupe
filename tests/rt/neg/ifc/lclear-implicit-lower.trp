(* negative test; lowering clearance in a context than the original *)

let val c = raisembox (`{alice,bob}`)        
in  if (true raisedTo`{alice}`) 
        then lowermbox (c, authority)
        else ()     
end
