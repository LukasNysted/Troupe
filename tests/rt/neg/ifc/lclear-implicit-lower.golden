2023-03-17T21:23:23.170Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
Runtime error in thread bce43100-0f90-4433-8eed-e8e1b33a6dd6@{}%{}
>> Cannot lower mailbox when the pc more sensitive than the mailbox clearance level
| current thread's pc level: {alice}
| mailbox clearance level: {}
